src/EventHandler.ts

```diff
@@ -20,11 +20,13 @@ export class Event {
         this.callbacks.push(callback);
     }
 
-    executeCallback(params?: any | any[]) {
+    executeCallback(params: any[]) {
```

Comment:
This is because I tried to allow the function `raiseEvent` to be able to receive multiple parameters, so I changed it to something like what's shown below.
```js
raiseEvent(name: string, ...params: any[]) {}
```
The `...params: any[]` will take in any number of parameters and store them in an array. That's why later in the `executeCallback` function I just make the parameter to be an array.